### ISSUE TITLE
Add support to install sqlsrv-5.9.0preview1 and pdo_sqlsrv-5.9.0preview1

### ISSUE THREAD

**Issue by shivammathur** [mlocati/powershell-phpmanager#70]
These should work
```ps1
Install-PhpExtension -Version 5.9.0 -Extension sqlsrv -MinimumStability devel -MaximumStability devel -Path C:\tools\php
```

```ps1
Install-PhpExtension -Version 5.9.0 -Extension pdo_sqlsrv -MinimumStability devel -MaximumStability devel -Path C:\tools\php
```

**Comment by mlocati** [comment]
I really don't understand when Windows DLLs are published under the `snaps` directory or under the `releases` directory.

I thought that only stable releases were published under the `releases` directory, but it's not so: for example, the `sqlsrv` packages with `devel` stability are sometimes published at https://windows.php.net/downloads/pecl/releases/sqlsrv/ and sometimes at https://windows.php.net/downloads/pecl/snaps/sqlsrv/

Anyway, #72 fixes this issue.

**Comment by shivammathur** [comment]
I'm also not sure, but I think if they are testing a pre-release version, they put it in snaps and after testing put it in releases directory as well, so always checking releases is a good idea.

Thanks for the fix.

**Comment by mlocati** [comment]
Fix included in new version  1.24.2
